eclipse/test/junit/saros/editor/internal/ContributionAnnotationManagerTest.java

```diff
@@ -232,19 +155,49 @@ public void testRemoveAnnotationsWhenUserLeaves() {
     while (!users.isEmpty()) {
       sessionListener.userLeft(users.remove(0));
 
-      assertEquals(
-          ContributionAnnotationManager.MAX_HISTORY_LENGTH * users.size(),
-          getAnnotationCount(model));
+      assertEquals(MAX_HISTORY_LENGTH * users.size(), getAnnotationCount(model));
     }
   }
 
+  @Test
+  public void testInsertAnnotationWithLengthGreaterOne() {
```

Comment:
I do not know if this is sufficient or to just introduce a test that combines multiple annotations lengths. Currently all other tests are only working with a length of 1.

Reply:
I added the test "testHistoryRemovelWithDifferentLengths"